pyproject.toml

```diff
@@ -82,7 +82,6 @@ test = [
 ]
 test-pyodide = [
   "pytest>=6",
-  "selenium<=4.25.0",  # unpin once >4.26.0 is available
```

Comment:
This just removes `"selenium<=4.25.0"` without replacing it with a generic `"selenium"`. Did you mean to do that? The tests still need to have selenium installed, right?

Reply:
Yeah, selenium is a dependency of `pytest-pyodide`, so I was thinking to just leave dependency/version management to them

Reply:
I guess that makes sense. I never know what to do about transitive dependencies. Perhaps if you use it directly (`import selenium`), then you should declare it as a dependency, just in case pytest-pyodide ever removes it or switches, but not if you don't use it directly.

Reply:
Hmm that's a good point. It is never used directly, so it should be fine.